libraries/radwordsprocessing/editing/mail-merge.md

```diff
@@ -95,15 +95,18 @@ __Example 4__ performs the mail merge operation over a previously defined templa
 
 ## Nested Mail Merge
 
-The nested mail merge functionality is supported from R1 2022. It allows you to merge data sources that contain nested data. For example, your business object can contain a list of other objects and this functionality allows accessing the properties of the underlying objects. In order to use the underlying objects, you need to  declare a group. Currently, the following group tags are supported. 
+The nested mail merge functionality is supported from R1 2022. It allows you to merge data sources that contain nested data. For example, your business object can contain a list of other objects and this functionality allows accessing the properties of the underlying objects. In order to use the underlying objects, you need to  declare a group. Currently, the following group tags are supported: 
 
 * BeginGroup/EndGroup 
 * TableStart/TableEnd
 * RangeStart/RangeEnd
 * GroupStart/GroupEnd
 
+Currently, all tags pairs work equally and more than one option exists in order to improve the readability of the documents.
 
-Currently, all tags work equally and more than one option exists in order to improve the readability of the documents. The below example demonstrates how you can use the nested mail merge. 
+>Exception: When a table row has only one cell, using the TableStart/TableEnd tags for the whole content of that cell will create a new row for each value. Every other pair of tags (BeginGroup/EndGroup, RangeStart/RangeEnd, GroupStart/GroupEnd) are interchangeable and will put the values on the same row inside that cell.
```

Comment:
...TableStart/TableEnd tags for the whole content...
should be:
...TableStart/TableEnd tags over the whole content...